### ISSUE TITLE
Questions regarding the falsification of experimental results from MCTrack on UniRTL

### ISSUE THREAD

**Issue by 08173021** [ydhcg-BoBo/RGB-T-Track#2]
Author, I reviewed your JSON file: you mixed four test set videos into the training set (30 + 4 = 34 videos). Furthermore, your paper explicitly states: 30 videos in the training set and 20 videos in the test set. Does this mean your experimental results are inflated?

Supporting materials:

Code check:

import json

Define file paths

paths = {

"test": "MCTrack-master/UniRTL_json/test.json",

"train": "MCTrack-master/UniRTL_json/train.json"

}

data = {}

for key, path in paths.items():

try:

with open(path, 'r', encoding='utf-8') as f:

print(f"Loading {key} data...")

data[key] = json.load(f)

print(f"✅ {key}.json loaded successfully! Contains {len(data[key].get('images', []))} images.")

except FileNotFoundError:

print(f"❌ Error: file {path} not found")

except json.JSONDecodeError:

print(f"❌ Error: {path} is not parsed into JSON.")

video_test = []
video_train = []

for video in data['test']['videos']:
video_test.append(video['file_name'])

for video in data['train']['videos']:
video_train.append(video['file_name'])

print('Duplicate video:')

for vid_test in video_test:
if vid_test in video_train:
print(vid_test)

img_test = []
img_train = []

for img in data['test']['images']:
img_test.append(img['vi_file_name'])

for img in data['train']['images']:
img_train.append(img['vi_file_name'])

img_repeat = 0
for img_test in img_test:
if img_test in `img_train:`
`img_repeat += 1`

# print(img_test)

print('Number of repeated images:', img_repeat)

Output:
Loading test data...

✅ test.json loaded successfully! Contains 12904 images.

Loading train data...

✅ train.json loaded successfully! Contains 14333 images.

Duplicate videos:

person_41019_5_564

person_41020_1_505

person_41021_3_641

person_41022_3_450
Duplicate images: 2160
